### PR TITLE
[IMP] website: redesign `s_features` snippet

### PR DESCRIPTION
**Pull request by anso-odoo**
---
This PR is handled by @chgo-odoo.

---

The goal of this PR is to improve the visual design of the default `s_features` snippet.

task-3657631

- Requires : https://github.com/odoo/design-themes/pull/826

-------------

| Before | After |
|--------|--------|
| Screenshot 2024-04-26 at 16 07 04 | image |
---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by chgo-odoo** [comment]
Hello @stefanorigano, could you have a look at this PR and the design-themes counterpart?
Thanks

File: addons/website_hr_recruitment/data/hr_job_demo.xml [hunk 1]
```diff
@@ -1298,37 +1363,50 @@
...
+                        </p>
+                    </div>
+                    <div class="row">
+                        <div class="col-lg-3">
+                            <div class="s_hr pt-4 pb32">
+                                <hr class="w-100 mx-auto"/>
+                            </div>
+                            <i class="fa fa-gift mb-3 rounded fs-5 bg-o-color-3"></i>
+                            <h3 class="h5-fs">Perks</h3>
+                            <p class="mb-0">A full-time position <br/>Attractive salary package.</p>
```
Comment (stefanorigano) [inline addons/website_hr_recruitment/data/hr_job_demo.xml:1385]: `mb-0`, why? 🤔

Beside conflicting with the theme's general settings (not bad per-se, but we should avoid unless really necessary), applying this class will bake the text "touch" the column edge when a bg-color is applied.

---
image

---

Also, the overall white-space management looks better without the class 👍 

| with `mb-0` | without `mb-0` |
|--------|--------|
| image | image |

File: addons/website_hr_recruitment/data/hr_job_demo.xml [hunk 2]
```diff
@@ -1298,37 +1363,50 @@
...
+                            </div>
+                            <i class="fa fa-futbol-o mb-3 rounded fs-5 bg-o-color-3"></i>
+                            <h3 class="h5-fs">Sport Activity</h3>
+                            <p class="mb-0">Play any sport with colleagues, <br/>the bill is covered.</p>
+                        </div>
+                        <div class="col-lg-3">
+                            <div class="s_hr pt-4 pb32">
+                                <hr class="w-100 mx-auto"/>
+                            </div>
+                            <i class="fa fa-coffee mb-3 rounded fs-5 bg-o-color-3"></i>
```
Comment (stefanorigano) [inline addons/website_hr_recruitment/data/hr_job_demo.xml:1407]: ;) 
```suggestion
                            <i class="fa fa-coffee mb-3 rounded fs-5 bg-o-color-3"/>
```

**Review by stefanorigano — CHANGES_REQUESTED** [review]
Good job here @chgo-odoo...
Just some minor remarks for you, well done  👍

File: addons/website/views/snippets/s_features.xml [hunk 3]
```diff
@@ -2,23 +2,36 @@
...
-                    <i class="fa fa-3x fa-gear rounded bg-o-color-1 m-3"/>
-                    <h3>First Feature</h3>
-                    <p>Tell what's the value for the <br/>customer for this feature.</p>
+                <div class="col-lg-4">
+                    <div class="s_hr pt-4 pb32">
+                        <hr class="w-100 mx-auto"/>
+                    </div>
+                    <i class="fa fa-paper-plane-o mb-3 rounded fs-5 bg-o-color-3" role="img"/>
+                    <h3 class="h5-fs">Reliability</h3>
+                    <p>Consistent performance and uptime ensure efficient, reliable service with minimal interruptions and quick response times.</p>
```
Comment (stefanorigano) [inline addons/website/views/snippets/s_features.xml:18]: The layout looks off when the image is aligned either left or right
| this PR | proposal |
|--------|--------|
| image | image | 


I propose to use the same approach brd is using in https://github.com/odoo/odoo/pull/172771, therefore something like:

```suggestion
                    <i class="fa fa-paper-plane-o mb-3 rounded fs-5 bg-o-color-3" role="img"/>
                    <div class="overflow-hidden>
                      <h3 class="h5-fs">Reliability</h3>
                      <p>Consistent performance and uptime ensure efficient, reliable service with minimal interruptions and quick response times.</p>
                    </div>
```
Be aware that you'll have to add some custom CSS, see https://github.com/odoo/odoo/pull/172771/files#diff-c3ce185eedf0ef674115e372d12e71b05054a2382643a33673ce6cefbf5350acR1

**Review by stefanorigano — CHANGES_REQUESTED** [review]
Thanks for handling this @chgo-odoo 👍 
Overall looks ok, I've just spot an functional issue when aligning the icon tho, more info in the comment, thanks!

File: addons/website/views/snippets/s_features.xml [hunk 4]
```diff
@@ -2,27 +2,52 @@
...
                 </div>
-                <div class="col-lg-4 pt32 pb32 text-center">
-                    <i class="fa fa-3x fa-leaf rounded bg-o-color-2 m-3"/>
-                    <h3>Third Feature</h3>
-                    <p>A small explanation of this great <br/>feature, in clear words.</p>
+                <div class="col-lg-4">
+                    <div class="s_hr pt-4 pb32">
+                        <hr class="w-100 mx-auto"/>
+                    </div>
+                    <i class="s_features_icon fa fa-flag-o float-start-o mb-3 rounded fs-5 bg-o-color-3" role="img"/>
```
Comment (stefanorigano) [inline addons/website/views/snippets/s_features.xml:36]: 😉
```suggestion
                    <i class="s_features_icon fa fa-flag-o float-start mb-3 rounded fs-5 bg-o-color-3" role="img"/>
```

File: addons/website/static/src/snippets/s_features/000.scss [hunk 5]
```diff
@@ -0,0 +1,10 @@
+.s_features[data-vcss='000'] {
+    .s_features_icon {
+        &.float-start {
+            margin-inline-end: var(--s_features_icon_margin, #{map-get($spacers, 3)}) !important;
+        }
+        &.float-end {
```
Comment (stefanorigano) [inline addons/website/static/src/snippets/s_features/000.scss:6]: Missing empty line between selectors ;)

File: addons/website_hr_recruitment/data/hr_job_demo.xml [hunk 6]
```diff
@@ -166,36 +166,49 @@
...
                             <p>12 days / year, including <br/>6 of your choice.</p>
                         </div>
-                        <div class="col-lg-3 text-center pt16 pb32">
-                            <i class="fa fa-2x fa-futbol-o rounded-circle text-bg-primary m-3"></i>
-                            <h3>Sport Activity</h3>
+                        <div class="col-lg-3">
+                            <div class="s_hr pt-4 pb32">
+                                <hr class="w-100 mx-auto"/>
+                            </div>
+                            <i class="fa fa-futbol-o mb-3 rounded fs-5 bg-o-color-3"></i>
```
Comment (stefanorigano) [inline addons/website_hr_recruitment/data/hr_job_demo.xml:202]: Don't forget `role="img`, other occurrences in modified lines as well ;)

File: addons/website_hr_recruitment/data/hr_job_demo.xml [hunk 7]
```diff
@@ -166,36 +166,49 @@
...
                             <p>A full-time position <br/>Attractive salary package.</p>
                         </div>
-                        <div class="col-lg-3 text-center pt16 pb32">
-                            <i class="fa fa-2x fa-bar-chart rounded-circle text-bg-primary m-3"></i>
-                            <h3>Trainings</h3>
+                        <div class="col-lg-3">
+                            <div class="s_hr pt-4 pb32">
+                                <hr class="w-100 mx-auto"/>
+                            </div>
+                            <i class="fa fa-bar-chart mb-3 rounded fs-5 bg-o-color-3"></i>
```
Comment (stefanorigano) [inline addons/website_hr_recruitment/data/hr_job_demo.xml:194]:
```suggestion
                            <i class="fa fa-bar-chart mb-3 rounded fs-5 bg-o-color-3"/>
```

**Review by stefanorigano — CHANGES_REQUESTED** [review]
Almost there 🙌
Just minors here and there, good job so far 😉

**Comment by chgo-odoo** [comment]
Hello @stefanorigano, this one should finally be ready to ship as well 😅

**Review by stefanorigano — APPROVED** [review]
Good to go, thanks @chgo-odoo 👍

**Comment by bso-odoo** [comment]
@robodoo rebase-ff r+

**Pull request closed without merging**